core/os/time.cpp

```diff
@@ -128,9 +131,9 @@ VARIANT_ENUM_CAST(Time::Weekday);
 	int64_t year = UNIX_EPOCH_YEAR_AD;                            \
 	Month month = MONTH_JANUARY;                                  \
 	uint8_t day = 1;                                              \
-	uint8_t hour = 0;                                             \
-	uint8_t minute = 0;                                           \
-	uint8_t second = 0;                                           \
+	int16_t hour = 0;                                             \
+	int16_t minute = 0;                                           \
+	int16_t second = 0;                                           \
```

Comment:
Since `split_ints` gives us `int`, we should likely just use `int` here, in case someone tries to parse `32770`.